docs/mine/lotus/miner-addresses.md

```diff
@@ -113,6 +113,11 @@ To set up a _control address_:
 
 Repeat this procedure to add additional addresses.
 
+5. You can Specify other actor to check control address:
+```sh
+lotus-miner --actor f01000 actor control list
+```
```

Comment:
````suggestion
5. You can Specify other actor to check control address:

```shell
lotus-miner --actor f01000 actor control list
```
````

- Add an empty line above codeblocks.
- Use `shell` instead of `sh`.